sm/mprv.h

```diff
@@ -3,36 +3,50 @@
 #include <stdbool.h>
 
 int copy1_from_sm(uintptr_t dst, const uint8_t *src);
-int copy8_from_sm(uintptr_t dst, const uint64_t *src);
-int copy64_from_sm(uintptr_t dst, const uint64_t *src);
+int copy_word_from_sm(uintptr_t dst, const uint64_t *src);
+int copy_block_from_sm(uintptr_t dst, const uint64_t *src);
 
 int copy1_to_sm(uint8_t *dst, uintptr_t src);
-int copy8_to_sm(uint64_t *dst, uintptr_t src);
-int copy64_to_sm(uint64_t *dst, uintptr_t src);
+int copy_word_to_sm(uint64_t *dst, uintptr_t src);
+int copy_block_to_sm(uint64_t *dst, uintptr_t src);
```

Comment:
just so `block` is clear here, can we define `struct block { size_t words[8] }` and have this function take a `struct block *`?